packages/api/src/services/subscriptions.ts

```diff
@@ -50,7 +50,7 @@ export const saveSubscription = async ({
   const result = await getRepository(Subscription).upsert(
     {
       name,
-      newsletterEmail,
+      newsletterEmail: { id: newsletterEmail.id },
```

Comment:
Is this a different issue?

Reply:
Yeah, I think this is not causing any issue just precaution